README.md

```diff
@@ -93,22 +94,44 @@ See [/examples/maildev](./examples/maildev) for a working example app using [Mai
 
 This plugin decorates fastify with a `mail` object containing the following methods:
 
-- `createMessage`: `function` - Generates a message from templates with context injected. 
+-  sendMail: 'function' - Calls `createMessage` to generate an message and uses [fastify-nodemailer](https://github.com/lependu/fastify-nodemailer) to send the generated email. 
   - Accepts the following arguments: 
-    - `recipients`: `array` - Array containing recipient[s] email address (`string`).
-    - `templates`: `string` - the relative path to the message's templates.
-    - `context`: `object` - Object containing context for the message.
-  - Returns: `object` with following properties:
-    - `to`: `array` - Array containing recipient[s] email address (`string`).
-    - `from`: `string` - The email address the email is to be sent from.
-    - `html`: `string` - The HTML of the email with context injected.
-    - `subject`: `string` - The subject of the email with context injected.
-
-- `sendMail`: `function` - Calls `createMessage` to generate an message and uses [fastify-nodemailer](https://github.com/lependu/fastify-nodemailer) to send the generated email. 
+    - message: 'object'
```

Comment:
after object on this line we could say that its a standard nodemailer object

Reply:
done